GliaWidgets/View/Chat/Message/Content/CustomCard/WebMessageCardView.swift

```diff
@@ -0,0 +1,168 @@
+import UIKit
+import WebKit
+
+private extension String {
+    static let readyState = "document.readyState"
+    static let bodyHeight = "document.body.scrollHeight"
+    static let bodyWidth = "document.body.scrollWidth"
+
+    static let separator = "@@"
+    static let gliaDomain = "gliaSdk"
+    static let js = """
+        function sendResponse(selectedText, selectedValue) {
+            window.webkit.messageHandlers.\(gliaDomain).postMessage(`${selectedText}\(separator)${selectedValue}`);
+        }
+    """
+    static let disableZooming = """
+        var meta = document.createElement('meta');
+        meta.name = 'viewport';
+        meta.content = 'width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no';
+        var head = document.getElementsByTagName('head')[0];
+        head.appendChild(meta);
+    """
+}
+
+protocol WebMessageCardViewDelegate: AnyObject {
+    func viewDidUpdateHight(_ view: WebMessageCardView)
+
+    func didSelectCustomCardOption(
+        _ view: WebMessageCardView,
+        selectedOption: HTMLMetadata.Option,
+        for messageId: String
+    )
+
+    func didSelectURL(
+        _ view: WebMessageCardView,
+        url: URL
+    )
+}
+
+final class WebMessageCardView: UIView {
+
+    private lazy var webView: WKWebView = {
+        let config = WKWebViewConfiguration()
```

Comment:
It would probably be beneficial to add comments describing what this configuration does.

Reply:
3e07ec159133ad017cb06c2141cb31e1ff1e431e